broadcast/broadcast.go

```diff
@@ -20,7 +21,7 @@ type Handler interface {
 // Message is a parsed message.
 type Message struct {
 	ID   nsq.MessageID
-	JSON map[string]interface{}
+	JSON json.RawMessage
```

Comment:
Ah, didn't notice this. I think we still want `json.RawMessage` to do the validation, so I've just changed it for now.